### PR TITLE
Add docs for `endBlock`

### PR DESCRIPTION
**Pull request by incrypto32**

### CODE REVIEW

File: website/pages/en/developing/creating-a-subgraph.mdx [hunk 1]
```diff
@@ -146,6 +147,8 @@ The important entries to update for the manifest are:
 
 - `dataSources.source.startBlock`: the optional number of the block that the data source starts indexing from. In most cases, we suggest using the block in which the contract was created.
 
+- `dataSources.source.endBlock`: the optional number of the block that the data source stops indexing at. When provided the dataSource stops indexing at the specified block.
```
Comment (azf20) [inline website/pages/en/developing/creating-a-subgraph.mdx:150]: is the block inclusive or exclusive? (i.e. in the example case does it index block 7175245 and stop, or does it stop before indexing 7175245?)

Comment (azf20) [reply website/pages/en/developing/creating-a-subgraph.mdx:150]: Additionally I think worth specifying the `specVersion` from which this is available

**Review by azf20 — CHANGES_REQUESTED** [review]
Thanks - two small asks!

**Comment by azf20** [comment]
Checking on this one @incrypto32?

**Comment by tamebadger** [comment]
what is the eta on this being merged ? is it safe to use endBlock now ?

**Review by azf20 — APPROVED** [review]
LGTM thanks!

**Pull request merged**